docs/pages/router/navigating-pages.mdx

```diff
@@ -65,28 +66,41 @@ To navigate to a route without the stack unwinding, you can use the `push` prop
 
 In contrast, the `replace` method substitutes the current route in the navigation stack with a new one, effectively replacing the current screen with the new one without adding to the stack.
 
-## Imperative navigation
-
-You may want to navigate from a global store when a user logs in or out. You can use the `router` object or `useRouter` hook to navigate imperatively (outside of React).
-
-```js
-import { router } from 'expo-router';
-
-export function logout() {
-  /* @info Navigate to <strong>/login</strong> */
-  router.replace('/login');
-  /* @end */
-}
-```
-
-The `router` object is immutable and contains the following functions:
-
-- **navigate**: `(href: Href) => void` Navigate to a route using the default behavior. You can provide a full path like **/profile/settings** or a relative path like **../settings**. Navigate to dynamic routes by passing an object like `{ pathname: 'profile', params: { id: '123' } }`.
-- **push**: `(href: Href) => void` Navigate to a route by always pushing onto the stack. You can provide a full path like **/profile/settings** or a relative path like **../settings**. Navigate to dynamic routes by passing an object like `{ pathname: 'profile', params: { id: '123' } }`.
-- **replace**: `(href: Href) => void` Navigate to a route by substituting the current route with a new one. This is useful for redirects.
-- **back**: `() => void` Navigate back to previous route.
-- **canGoBack**: `() => boolean` Returns `true` if a valid history stack exists and the `back()` function can pop back.
-- **setParams**: `(params: Record<string, string>) => void` Update the query params for the currently selected route.
+To navigate, you can provide a full path like `/profile/settings`, a relative path like `../settings` or by passing an object like `{ pathname: 'profile', params: { id: '123' } }`.
```

Comment:
```suggestion
To navigate, you can provide a full path (`/profile/settings`), a relative path (`../settings`), or by passing an object (`{ pathname: 'profile', params: { id: '123' } }`).
```

Put the examples in brackets so that it is easier to read.